htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -348,6 +348,10 @@
 		if (!empty($conf->service->enabled) && ($object->element == 'facturerec' || $object->element == 'invoice_supplier_rec')) {
 			echo '<div class="divlinefordates"><br>';
 			echo $langs->trans('AutoFillDateFrom').' ';
+			if ($conf->global->INVOICE_REC_DATE_TO_YES) {
```

Comment:
`if (!empty($conf->global->INVOICE_REC_DATE_TO_YES)) {`
if not defined, it will make warning with php8